corehq/apps/change_feed/data_sources.py

```diff
@@ -11,7 +12,7 @@
 FORM_SQL = 'form-sql'
 CASE_SQL = 'case-sql'
 SMS = 'sms'
-LEDGER = 'ledger'
+LEDGER_NEW = 'ledger-new'
```

Comment:
:+1: will go with `ledger-v2` since they are both stored in SQL

Also just to verify that there's no reason to create 2 separate kafak topics